Automattic/wp-calypso
# Add Do Not Sell Dialog component to @automattic/privacy-toolset

#### Proposed Changes

* Create a Do Not Sell Dialog component

<img width="777" alt="CleanShot 2022-12-01 at 22 25 21@2x" src="https://user-images.githubusercontent.com/8419292/205214578-e42daa28-5bec-45ea-8739-98d51fc0665c.png">


#### Testing Instructions

* Checkout the changes
* Run `yarn workspace @automattic/privacy-toolset run storybook`
* Storybook should automatically open in your browser
* Go to the path `?path=/story/do-not-sell-dialog--default`
* The Do Not Sell Dialog should be styled as per the design above

#### Pre-merge Checklist


#### After-merge Checklist

- [x] Publish update to @automattic/privacy-toolset on npmjs.com
